sdk/ml/azure-ai-ml/azure/ai/ml/_utils/_asset_utils.py

```diff
@@ -652,6 +652,33 @@ def _create_or_update_autoincrement(
     return result
 
 
+def _get_next_version_from_container(
```

Comment:
def _create_or_update_autoincrement will be replaced by this when model and enironment is also ready.